recipe/meta.yaml

```
@@ -1,19 +1,19 @@
 {% set name = "pyspark" %}
-{% set version = "3.5.4" %}
+{% set version = "4.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: 1c2926d63020902163f58222466adf6f8016f6c43c1f319b8e7a71dbaa05fc51
+    sha256: 38db1b4f6095a080d7605e578d775528990e66dc326311d93e94a71cfc24e5a5
   # get the LICENSE from the GH repo
   - url: https://raw.githubusercontent.com/apache/spark/refs/tags/v{{ version }}/LICENSE
     sha256: 80209caf5bfbc82ad9163d9a42d7be5cbdcbf26e4e2b13666cdc11e4729ab50d
 
 build:
-  number: 1
+  number: 0
   # pyarrow isn't available on s390x.
   skip: True  # [py<38 or s390x]
```

```suggestion
  skip: True  # [py<39]
```
Since this version of pyspark requires at least python 3.9 or later we can skip builds for lesser versions.